easybuild/easyconfigs/p/Platanus/Platanus-1.2.4-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Platanus'
+version = '1.2.4' 
+
+homepage = 'http://platanus.bio.titech.ac.jp/'
+description = """PLATform for Assembling NUcleotide Sequences"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://platanus.bio.titech.ac.jp/?ddownload=150']
```

Comment:
@boegel  yes it does. But this is just by chance, as their webserver don't care of the end of the url:)
It could be a good idea to have a way to specify the sources, and an optional destination filename, because if I put ?download=150 in the sources, eb doesn't recognize the file type.